cmd/lakectl/cmd/config.go

```diff
@@ -34,9 +37,15 @@ var configCmd = &cobra.Command{
 		}{
 			{Key: "credentials.access_key_id", Prompt: &promptui.Prompt{Label: "Access key ID"}},
 			{Key: "credentials.secret_access_key", Prompt: &promptui.Prompt{Label: "Secret access key", Mask: '*'}},
-			{Key: "server.endpoint_url", Prompt: &promptui.Prompt{Label: "Server endpoint URL", Validate: func(rawURL string) error {
-				_, err := url.ParseRequestURI(rawURL)
-				return err
+			{Key: "server.endpoint_url", Prompt: &promptui.Prompt{Label: "Server endpoint", Validate: func(rawURL string) error {
+				u, err := url.ParseRequestURI(rawURL)
+				if err != nil {
+					return err
+				}
+				if u.Path != "" {
+					return fmt.Errorf("%w: do not specify endpoint path", ErrInvalidEndpoint)
```

Comment:
I think that it shouldn't return an error but instead add a guidance that instructs the user to provide the server's root endpoint: `Root sever endpoint URL` or something of this sort